CHANGELOG.md

```diff
@@ -24,17 +24,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+- The public `ExprValue` construction methods are migrated from `ExprValueFactory` to `ExprValue` interface as public static methods & states.
```

Comment:
What is "state" here?  If that's what is called class "fields" in Java or "properties" in Kotlin, then, even though some of them are involved, it would be sufficient if this changelog entry just says "methods".

In my opinion, this whole sentence would better be merged into the one in "Deprecated" section -- to reduce duplication and the need for the reader to track and connect these two pieces together.

Reply:
I mean "properties" here. I will move this description to "Deprecated" section